bin/interpreter.sh

```diff
@@ -47,6 +47,24 @@ fi
 
 . "${bin}/common.sh"
 
+INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
+
+
+if [[ -z "${ZEPPELIN_INTP_MEM}" ]]; then
+  export ZEPPELIN_INTP_MEM="${ZEPPELIN_INTP_DEFAULT_MEM}"
+fi
+
+INTERPRETER_SPECIFIC_MEM_VARIABLE=`echo $INTERPRETER_ID | awk '{print toupper($0)"_INTP_MEM" }'`
```

Comment:
I agree. however, i think until interpreter setting page get's such feature, this PR's approach can be useful. And once interpreter setting page have mem configure feature, this env variable can provides default value.